src/renderer/components/SyncView/SyncFooter.js

```diff
@@ -23,7 +25,9 @@ const SyncFooter = () => {
       <Toolbar>
         <div className={cx.titleBar}>
           <Typography component='h2' className={cx.title}>
-            {encryptionKey}
+            {encryptionKey
+             ? `${encryptionKey.slice(0, visibleKeyLength)}${'*'.repeat(encryptionKey.length - visibleKeyLength)}`
```

Comment:
I don't think we need to fill `*` to the length of the key. It could cause UI issues (string too long) especially if the use has increased text size. I see in [mobile](https://github.com/digidem/mapeo-mobile/blob/c7df1fd798a5587d6444cde9127752e1b238fddc/src/frontend/screens/SyncModal/SyncView.js#L140) it was pretty random, I just chose 10 `*`, maybe let's do that here for good measure. At some point the encryption key could grow and we don't want a 50-char string here.

Reply:
Totally agree :) fixed in https://github.com/digidem/mapeo-desktop/pull/419/commits/ddd4d386b7b23f58b7a360f102825ada171a0f79